test/visualize/test_visualize.py

```diff
@@ -174,6 +198,18 @@ def test_parameters():
         # test plotting of lists
         pypesto.visualize.parameters([result_1, result_2])
 
+    @staticmethod
+    def test_parameters_with_nan_inf():
+        # create the necessary results
+        result_1 = create_optimization_result_nan_inf()
+        result_2 = create_optimization_result_nan_inf()
+
+        # test a standard call
+        pypesto.visualize.parameters(result_1)
+
+        # test plotting of lists
+        pypesto.visualize.parameters([result_1, result_2])
+
```

Comment:
great to have added tests already! can you add one more, which tests delete_nan_inf directly? just create e.g. function values ``[42, 1.5, np.nan, 67.01, np.inf]`` and then ``assert [42, 1.5, 67.01] == delte_nan_inf(...)``

Reply:
> looks great. @paulstapor fine for you?

gimme a bit time. Will look at it today :)
First look seems nice, tests still need to pass.